visual_egomotion:
  ros__parameters:
    visual_topic: /aruco_detector/landmarks
    use_sim_time: true

    ref_frame: vo_ref
    base_frame: base_link
    cam_frame: Orca/camera_front

    # Sliding-window
    max_age_sec: 15.0 
    window_size: 20

    publish_tf: false
    pub_rate_hz: 20.0
    lost_timeout_sec: 15.0
    huber_threshold_deg: 2.5
    gate_threshold_deg: 15.0

    have_ref: false
    bool have_last_ref_base_: false